ingest-app/src/cmr/ingest/services/ingest_service/util.clj

```diff
@@ -12,8 +12,6 @@
    [cmr.ingest.data.provider-acl-hash :as pah]
    [cmr.message-queue.queue.queue-protocol :as queue-protocol]
    [cmr.oracle.connection :as conn]
-   [cmr.redis-utils.redis :as redis]
```

Comment:
did you mean to remove this?

Reply:
yes, it was not called in the file (both the Code GUI thought so by underlining it, and I searched the file).